CampaignConfig/mdc2020_primary.ini

```diff
@@ -0,0 +1,65 @@
+[campaign]
+experiment = mu2e
+poms_role = production
+name = mdc2020_primary_081924
+campaign_stage_list = generation_par, generation
+
+[campaign_defaults]
+vo_role=Production
+campaign_keywords={"release_v_dts": "ag", 
+		   "release_v_stops": "p",
+		   "stopped_type": "Muminus",
+		   "primary_name": "DIOtail",
```

Comment:
We need to follow my comment or design a new way to include the mom start value in the name. The "Tag" parameter was used to do this previously